packages/marketplace/src/services/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const API_VERSION = '31-01-2020'
+export const API_VERSION = '2020-01-31'
```

Comment:
Why do you need to change this?

Reply:
I got below message when I hit API at https://dev.platform.reapit.cloud/negotiators

> dateTime: "2020-05-25T14:51:28.4105056Z"
> description: "HTTP resource requires that requests are issued with a 'api-version' header containing a UTC version date. Format: yyyy-MM-dd'"
> statusCode: 400

So I think "yyyy-MM-dd" is the correct format for API_VERSION